### ISSUE TITLE
[a11y]: MultiSelect - Accessibility issue - The ARIA attribute "aria-autocomplete" is in conflict with the corresponding HTML attribute "autocomplete" 

### ISSUE THREAD

**Issue by omkarajagunde** [carbon-design-system/carbon#17267]
### Package

@carbon/react

### Browser

Chrome, Safari, Firefox, Edge

### Operating System

MacOS

### Package version

^1.51.0

### React version

^18.2.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

<img width="1439" alt="Screenshot 2024-08-26 at 12 06 16 PM" src="https://github.com/user-attachments/assets/5497e81f-3acd-4ebc-bb59-5268fed3080d">

### Reproduction/example

NA

### Steps to reproduce

On the react storybook - https://react.carbondesignsystem.com/?path=/docs/components-multiselect--overview 
if I try to run **IBM Equal Access Accessibility Checker** the below Violation is seen - 
The ARIA attribute "aria-autocomplete" is in conflict with the corresponding HTML attribute "autocomplete"

### Suggested Severity

None

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Duplicate of https://github.com/carbon-design-system/carbon/issues/17181
